### PR TITLE
content: link updates for dev portal

### PR DESCRIPTION
**Pull request by kendallstrautman**
This PR updates links to redirected paths so they may resolve properly in the dev portal context.

### CODE REVIEW

File: website/content/vagrant-cloud/support.mdx [hunk 1]
```diff
@@ -33,7 +33,7 @@ it provides, there may be information lacking in the documentation.
 Vagrant Cloud documentation is open source.
 If you'd like to improve or correct the documentation,
 please submit a pull-request to the
-[Vagrant project on GitHub](https://github.com/hashicorp/vagrant/tree/main/website/pages/docs/vagrant-cloud/).
+[Vagrant project on GitHub](https://github.com/hashicorp/vagrant/tree/main/website/pages/vagrant-cloud/).
```
Comment (tunzor) [inline website/content/vagrant-cloud/support.mdx:36]: Think the directory structure has changed a bit -  both URLs lead to a 404. This new path seems to work
```suggestion
[Vagrant project on GitHub](https://github.com/hashicorp/vagrant/tree/main/website/content/vagrant-cloud/).
```

**Review by tunzor — CHANGES_REQUESTED** [review]
Looks good! Just one change for a broken link

File: website/content/vagrant-cloud/support.mdx (hunk 1 above)
Comment (kendallstrautman) [reply website/content/vagrant-cloud/support.mdx:36]: Good catch!

**Review by tunzor — APPROVED** [review]

**Review by ashleemboyer — APPROVED** [review]
LGTM! 👍🏻

**Pull request merged**